rust/azure_iot_operations_connector/src/data_transformer.rs

```diff
@@ -1,51 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
My bad, I thought I had already removed this

Reply:
no worries!